tasks/main.yml

```diff
@@ -1,2 +1,31 @@
 ---
-# tasks file for skeleton
+- name: Install systemd-resolved
+  ansible.builtin.package:
+    name:
+      - systemd-resolved
+
+- name: Enable and start systemd-resolved
+  ansible.builtin.service:
+    enabled: true
+    name: systemd-resolved.service
+    # We must start the service to populate the files in
+    # /run/systemd/resolve/, which we need when we create a symlink
+    # below.
+    state: started
+
+- name: Ensure resolvconf is not installed
+  ansible.builtin.package:
+    name:
+      - resolvconf
+    state: absent
+
+- name: Create /etc/resolv.conf symlink
+  ansible.builtin.file:
+    # Do not apply any group, owner, or mode changes to src.
```

Comment:
Since we are not setting those parameters I am unclear on what this comment is implying happens based on the [documentation](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_module.html#parameter-follow).

Reply:
Please see commit 476fe67ca4f8f49260f2ac8e5804f96a35deaeb2.

Reply:
Ahh thanks. My reading of the documentation definitely does not reflect your observed behavior.